drools-parser/src/main/java/org/drools/parser/DRLParserError.java

```diff
@@ -0,0 +1,54 @@
+package org.drools.parser;
+
+public class DRLParserError {
```

Comment:
This class is similar to `org.drools.drl.parser.DroolsParserException`. If this project depends on `drools-drl-parser`, we may use the `org.drools.drl.parser.DroolsParserException`. But we don't want to depend on `drools-drl-parser` at the moment? (= because we want keep the project simple at the moment)